geopmdpy/geopmdpy/pio.py

```diff
@@ -700,8 +607,7 @@ def reset():
     cleared, any batch servers that had been started will be stopped,
     and all registered IOGroups will be reset.
     """
-    global _dl
-    _dl.geopm_pio_reset()
+    lib.geopm_pio_reset()
```

Comment:
where is lib coming from?

Reply:
Fixed that bad change in the latest push